metafix/src/main/resources/log4j.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE log4j:configuration SYSTEM "http://logging.apache.org/log4j/1.2/apidocs/org/apache/log4j/xml/doc-files/log4j.dtd">
+
+<log4j:configuration>
+    <appender name="stdout" class="org.apache.log4j.ConsoleAppender">
+        <layout class="org.apache.log4j.PatternLayout">
+            <param name="ConversionPattern"
+                value="%-5p [%t] [%c{1}] %m%n" />
+        </layout>
+    </appender>
+
+    <root>
+        <priority value="INFO" />
```

Comment:
Simplify changing the log level at runtime:

```suggestion
        <priority value="${env:METAFIX_LOG_LEVEL:-INFO}" />
```

Reply:
How is the variable set? `export METAFIX_LOG_LEVEL=DEBUG; bin/metafix-runner  $pathTo.flux` didnt' work. Same for `export METAFIX_LOG_LEVEL="DEBUG"`.

Reply:
hmpf, no this works , BUT it's always on DEBUG level regardless what I set .... oh , logging is so complicated ...

Reply:
Oh, sorry, I overlooked that we're on Log4j 1.x which doesn't seem to support [property substitution](https://logging.apache.org/log4j/2.x/manual/configuration.html#property-substitution).

Reply:
No, it doesn't work. It's effectively an invalid/empty log level.

Reply:
Okay, apparently it does support property substitution, it's just [not readily documented](https://logging.apache.org/log4j/1.x/apidocs/org/apache/log4j/PropertyConfigurator.html). However, I wasn't able to figure out how to supply a default value in the configuration file. So it gets a little more complicated:

```xml
<priority value="${org.metafacture.metafix.logLevel}" />
```

```diff
diff --git metafix-runner/build.gradle metafix-runner/build.gradle
index 1179b5c..ab71abb 100644
--- metafix-runner/build.gradle
+++ metafix-runner/build.gradle
@@ -39,11 +39,15 @@ dependencies {
 application {
   mainClass = 'org.metafacture.runner.Flux'
 
+  applicationDefaultJvmArgs = [
+    "-Dorg.metafacture.metafix.logLevel=INFO"
+  ]
+
   if (project.hasProperty('profile')) {
     def file = project.getProperty('profile') ?: project.name
     def depth = project.hasProperty('profile.depth') ? project.getProperty('profile.depth') : 8
 
-    applicationDefaultJvmArgs = [
+    applicationDefaultJvmArgs += [
       "-XX:FlightRecorderOptions=stackdepth=${depth}",
       "-XX:StartFlightRecording=dumponexit=true,filename=${file}.jfr,settings=profile"
     ]
```

Then run with `JAVA_OPTS=-Dorg.metafacture.metafix.logLevel=DEBUG`.